### PR TITLE
Update nvim-workbench.vim

### PR DESCRIPTION
**Pull request by Vanderscycle**
Fixed the highlighting problem. :noh turns off highlighting until the next search. Essentially I piped the command.

Fixes Issues #4 

Works on my config but unsure if there are issues on different configs.

### CODE REVIEW

File: plugin/nvim-workbench.vim [hunk 1]
```diff
@@ -8,13 +8,13 @@ endfunction
 " Toggle checkbox
 " - [ ] testing -> - [x] testing
 " - [x] testing -> - [ ] testing
-nnoremap <expr> <silent> <Plug>WorkbenchToggleCheckbox <SID>SearchCheck() ? ':s/\[.*\]/\[x\]<CR>': ':s/\[x\]/\[ \]<Enter>' 
+nnoremap <expr> <silent> <Plug>WorkbenchToggleCheckbox <SID>SearchCheck() ? ':s/\[.*\]/\[x\]<CR>': ':s/\[x\]/\[ \]<Enter>/<bar>noh<CR>'
```
Comment (weizheheng) [inline plugin/nvim-workbench.vim:11]: Does this only works if `SearchCheck()` returns false? We might want to add to the first part as well? When `SearchCheck()` returns true.

**Review by weizheheng — CHANGES_REQUESTED** [review]
Thank you so much for spending time to make this PR. Left a small comment, and I will test it locally during the weekend. If everything works perfectly, then I will merge it.

File: plugin/nvim-workbench.vim (hunk 1 above)
Comment (Vanderscycle) [reply plugin/nvim-workbench.vim:11]: No, it doesn't work actually.

**Comment by Vanderscycle** [comment]
I think my pr solves the main highlighting issue but not the edge cases. I'll need to delve deeper into vim and learn a bit more.

**Comment by weizheheng** [comment]
Hi @Vanderscycle, sorry for the late reply, it has been a busy week at work. I have received a few issues regarding the same problem, so I have decided to fix it. I think the current code should fix the highlighting problem. I have learned a lot throughout the process as well, you can take a look at the recent commits on how I am trying to solve it. I am using some example I found online as well. Credit to (https://gist.github.com/olmokramer/feadbf14a055efd46a8e1bf1e4be4447)

Also, I have combined the `adding and toggling` function into one, so that you can call the below mapping and it will do the job, either adding or toggling the checkbox. Of course, that old function is still there, and you can still call it without having the highlighting problem now.
```vim
nmap <leader><CR> <Plug>WorkbenchToggleCheckbox
```

If you don't mind, can you try updating the plugin and see if it works. Anyway, really appreciate you spending time trying to fix this :) Have a great day.

**Comment by Vanderscycle** [comment]
Sounds good, I'll give it a look.

**Pull request closed without merging**